bundle/src/main/java/com/adobe/acs/commons/wcm/impl/SiteMapServlet.java

```diff
@@ -139,6 +160,39 @@ protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse r
         }
     }
 
+    private Collection<Resource> getAssetFolders(Page page, ResourceResolver resolver) {
+        Map<String, Resource> allAssetFolders = new TreeMap<String, Resource>();
+        ValueMap properties = page.getProperties();
+        String[] configuredAssetFolderPaths = properties.get(damAssetProperty, String[].class);
+        if (configuredAssetFolderPaths != null) {
+            for (String configuredAssetFolderPath : configuredAssetFolderPaths) {
```

Comment:
For each configured DAM Asset folder, find all subfolders and add to the list as well.  Code filters out duplicates so that if a user specifies two folders where one is a child of the other the assets are not added to the sitemap twice.